packages/components/src/InputDate/InputDate.stories.mdx

```diff
@@ -0,0 +1,102 @@
+import { ArgsTable, Canvas, Meta, Story } from "@storybook/addon-docs";
+import { InputDate } from "@jobber/components/InputDate";
+import { useState } from "react";
+import { ExampleWithHooks } from "mdxUtils/ExampleWithHooks";
+
+<Meta title="Components/InputDate" component={InputDate} />
```

Comment:
In Atlantis Web this is in the `PATTERNS` section.  I created that section in another PR for `Page` and iirc this should be able to just be:

```
<Meta title="Patterns/InputDate" component={InputDate} />
```

to make that heading show up